app/views/assignments/_form.html.slim

```diff
@@ -1,5 +1,6 @@
 = simple_form_for(@assignment) do |f|
   = simple_error_notice f
+  = f.input :user_id, as: :hidden
```

Comment:
also for my understanding related to my question above, when we add here the hidden field, isn't the id passed with current "client" ?